src/components/Page/README.md

```diff
@@ -186,12 +186,20 @@ Use for detail pages, which should have pagination and breadcrumbs, and also oft
 <Page
   breadcrumbs={[{content: 'Products', url: '/products'}]}
   title="Jar With Lock-Lid"
+  titleMetadata={<Badge>Draft</Badge>}
   primaryAction={{content: 'Save', disabled: true}}
   secondaryActions={[{content: 'Duplicate'}, {content: 'View on your store'}]}
+  actionGroups={[
+    {
+      title: 'Promote',
+      actions: [{content: 'Share on Facebook', onAction: () => {}}],
+    },
+  ]}
   pagination={{
     hasPrevious: true,
     hasNext: true,
   }}
+  separator
```

Comment:
Ah, I see you added a separator here. That probably accounts for that part of the failing VRT.